eng/native/version/copy_version_files.sh

```diff
@@ -0,0 +1,21 @@
+__VersionFolder="$(cd "$(dirname "$0")"; pwd -P)"
```

Comment:
nit: since we are using bash syntax, we can add shebang in this file as well.
```suggestion
#!/usr/bin/env bash

__VersionFolder="$(cd "$(dirname "$0")"; pwd -P)"
```
(it normally doesn't affect eval, source or standalone scenarios; only helps when the executing environment is not using bash)